js-ext-core/src/accountkey/accountkey.ts

```diff
@@ -1,7 +1,7 @@
 
-import { FieldSet, FieldTypeAccountKeyList, FieldTypeCompressedPubKey, FieldTypeUint32, FieldTypeUint8, FieldTypeWeightedPublicKeys } from "../field";
-import { AccountKeyType, HexStr, RLP } from "../util";
-import { convertKeysToRLP, decodeObjectFromRLP, GetHexlifyRLP } from "../util/rlp";
+import { FieldSet, FieldTypeAccountKeyList, FieldTypeCompressedPubKey, FieldTypeUint32, FieldTypeUint8, FieldTypeWeightedPublicKeys } from "../field/index.js";
+import { AccountKeyType } from "../util/const.js";
```

Comment:
.js extension is added since esm don't accept directory import, without extension import